### DISCUSSION TITLE
How do you choose where the file is going to be located?

### DISCUSSION THREAD

**Discussion by 1nfiniteZest** [crunchy-labs/crunchy-cli#373] [Q&A] [answered]
Im hoping to download some episodes onto a hard drive. Is this possible? Sorry, im new at this.

**Comment by bytedream** [comment, marked as answer]
You can use the [`-o` flag](https://github.com/crunchy-labs/crunchy-cli#user-content-download-output) for this. E.g. `-o /drive/myfolder` would download everything into `/drive/myfolder`

**Reply by 1nfiniteZest** [reply]
Thank you so much!
